src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -663,6 +663,41 @@ struct accel_deadlock_status
   }
 };
 
+struct dtbo_id
+{
+  using result_type = query::dtbo_id::result_type;
+
```

Comment:
`using slot_id_type = query::dtbo_id::slot_id_type;`

Reply:
@soren I changed struct from dtbo_id to dtbo_path because of libdfx library changes and handled all the review comments, please review.